providers/docker/tox.ini

```diff
@@ -0,0 +1,67 @@
+[tox]
+envlist = py35,py36,py38,py310
+skip_missing_interpreters = true
+skipsdist=True
+
+[testenv]
+commands =
+    pip -q install -e ../../checkbox-ng
+    - {envbindir}/python3 ../resource/manage.py develop
+    - {envbindir}/python3 ../base/manage.py develop
+    {envbindir}/python3 manage.py validate
```

Comment:
Is it normal the `manage.py test` command is not run for this provider?

Reply:
Actually `manage.py test` is not run for two providers, docker and tpm2.

- Docker: too many shellcheck errors are reported by manage.py test, unless the original maintainer (GarryW) helps us to fix them all, I'd recommend to just perform the usual provider validation on, this one.

- TPM2: problem with this one is the `bin/` dir which contains upstream tpm2-tools scripts copies. Again let's just run the provider validation.